hawkular-alerts-engine/src/main/java/org/hawkular/alerts/engine/impl/PartitionManagerImpl.java

```diff
@@ -183,6 +186,19 @@ public boolean isDistributed() {
         return distributed;
     }
 
+    @Override
+    public Map<String, String> getStatus() {
+        if (distributed) {
+            /*
+                Members are dynamic, so on each status request this info is refreshed
+             */
+            status.put("members", cacheManager.getMembers().stream()
```

Comment:
I prefer the C syntax for comments, it makes comment more clear for me :-).
Having this extra space makes me to pay better attention to it.